syntax = "proto3";

option cc_generic_services = true;

message LoginRequest {
  string username = 1;
  string password = 2;
}
message LoginResponse {
  bool isLoginSuccessfully = 1;
}

message RegRequest {
  string username = 1;
  string password = 2;
  int32 age = 3;
  int32 gender = 4;
}
message RegResponse {
  bool isRegSuccessfully = 1;
}

service CheckServiceRpc {
  rpc login(LoginRequest) returns (LoginResponse);
  rpc reg(RegRequest) returns (RegResponse);
}

